README.md

```diff
@@ -152,10 +172,57 @@ dashboard. On your development environment,
 make sure to use test keys. If you don't, you're
 going to be charged real money!
 
-#### Testing
+### Stripe keys setup
+
+#### Get Stripe keys:
+
+Go to [Stripe](https://stripe.com), create an account or just log in with you already have one. Access the stripe dashboard and copy both publishable and secret keys.
+> make sure to use test keys. If you don't, you're
+going to be charged real money!
+
+![get Stripe keys](https://user-images.githubusercontent.com/31708472/157132661-79bf89a0-13cb-4860-9793-a40bb3229bfb.png)
+
+
+ #### Configure the .env file:
+ 
+ Then after retrieving both keys copy them into your .env file on these lines:
+ ```
+ export STRIPE_API_KEY='REPLACE' # use your test private key from your stripe account
+ export STRIPE_API_PUBLIC='REPLACE' # use your test public key from your stripe account
+ ```
+
+### Start project:
```

Comment:
I'm confused on this ordering. They haven't created a nonprofit or user yet so there's no real point to running `bin/rails server`; they won't be able to do anything.